spec/spec.md

```diff
@@ -1070,6 +1072,41 @@ outcomes in the database.
        "Hib3",   1996,    50,     "AGO",       "Angola",          5000,     1000,      NA,    5670
        "Hib3",   1997,    50,     "AGO",       "Angola",          6000,     1200,      NA,    5870
        
+       
+## POST /modelling-groups/{modelling-group-id}/responsibilities/{touchstone-id}/{scenario-id}/estimate-set/
```

Comment:
Would prefer it if this were plural: `estimate-sets`. This has been the pattern throughout, because then you can have:

* `GET /foos/` (Gets all the foos)
* `GET /foos/45` (Gets one particular foo - also use this URL pattern, with other verbs for modifying, deleting, etc.)
* `POST /foos/` (Add a new foo to the list)